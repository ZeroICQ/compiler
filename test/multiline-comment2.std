(2:0)	EOF	[end of file]	"EOF"
