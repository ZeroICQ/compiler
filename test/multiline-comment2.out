(1:0)	comment	[]	"{
}"
(2:0)	EOF	[]	" "
